cmd/minikube/cmd/delete.go

```diff
@@ -295,6 +303,48 @@ func deleteProfile(ctx context.Context, profile *config.Profile) error {
 	return nil
 }
 
+func unpauseIfNeeded(profile *config.Profile) error {
+	// there is a known issue with removing kicbase container with paused containerd containers inside
+	// unpause it before we delete it
+	if profile.Config.KubernetesConfig.ContainerRuntime != "containerd" {
```

Comment:
crio also has this problem., we can check if Not Docker

Reply:
fixed